README.md

```diff
@@ -78,6 +84,49 @@ We use GitHub issues for tracking bugs and feature requests. Please use these co
 
 Please see the [contribution guidelines](.github/CONTRIBUTING.md).
 
+<!-- BEGIN_TF_DOCS -->
+## Modules
+
+No modules.
+
+## Inputs
+
+| Name | Description | Type | Default | Required |
+|------|-------------|------|---------|:--------:|
+| <a name="input_ocean_vng_name"></a> [ocean\_vng\_name](#input\_ocean\_vng\_name) | Enter a name for the virtual node group. | `string` | `null` | yes |
+| <a name="input_ocean_id"></a> [ocean\_id](#input\_ocean\_id) | The Ocean cluster identifier. Required for Launch Spec creation. | `string` | `null` | yes |
+| <a name="input_headrooms_num_of_units"></a> [headrooms\_num\_of\_units](#input\_headrooms\_num\_of\_units) | The number of units to retain as headroom, where each unit has the defined headroom CPU, memory. | `number` | `null` | no |
```

Comment:
change to autoscale_headrooms_

Reply:
appended autoscale_ to update variable names.